staff/didac-torres/blog/entries.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Blog</title>
+</head>
+
+<body>
+    <header>
+        <div class="title">
```

Comment:
@didactor91 no divs allowed !This is a semantic version. Please remove them all. Thanks.

Reply:
Done!